### ISSUE TITLE
Want to train some actions yourself

### ISSUE THREAD

**Issue by shankaijie** [tensorflow/tensorflow#58652]
Want to train some actions yourself, compile the file of the boss and report an error

bug position：
converter = tf.lite.TFLiteConverter.from_keras_model(model)
converter.optimizations = [tf.lite.Optimize.DEFAULT]
tflite_model = converter.convert()

bug information：Some of the operators in the model are not supported by the standard TensorFlow Lite runtime. If those are native TensorFlow operators, you might be able to use the extended runtime by passing --enable_select_tf_ops, or by setting target_ops=TFLITE_BUILTINS,SELECT_TF_OPS when calling tf.lite.TFLiteConverter(). Otherwise, if you have a custom implementation for them you can disable this error with --allow_custom_ops, or by setting allow_custom_ops=True when calling tf.lite.TFLiteConverter(). Here is a list of builtin operators you are using: ADD, DIV, EXPAND_DIMS, FLOOR_DIV, FULLY_CONNECTED, GATHER, MAXIMUM, MUL, PACK, REDUCE_MAX, RESHAPE, SOFTMAX, SQRT, SQUEEZE, STRIDED_SLICE, SUB, SUM. Here is a list of operators for which you will need custom implementations: BroadcastTo, Size.

add： converter.allow_custom_ops=True

final：16:23converter = tf.lite.TFLiteConverter.from_keras_model(model)
converter.optimizations = [tf.lite.Optimize.DEFAULT]
converter.allow_custom_ops=True
tflite_model = converter.convert()

This compilation is successful, but the tflite file will not run on the AS,why?help me

**Comment by tiruk007** [comment]
@shankaijie 
Can you please fill the issue [template](https://github.com/tensorflow/tensorflow/issues/new/choose) and give a reproducible  code snippet or colab gist for replicate  the issue reported here.

Thank you!
